recipe/meta.yaml

```
@@ -31,96 +32,71 @@
 {% set tests = "test_parallel_for test_tbb_header conformance_version test_flow_graph test_arena_constraints" %}
 {% set ctest_regex = "^(%s)$" % (tests.replace(' ', '|')) %}
 
-{% set tbb4py_build_dir = "build_py${PY_VER}h${PKG_HASH}" %}  # [not win]
-{% set tbb4py_build_dir = "build_py%PY_VER%h%PKG_HASH%" %}    # [win]
-
 package:
-  name: tbb
+  name: {{ name }}-split
   version: {{ version }}
 
 source:
   url: https://github.com/oneapi-src/oneTBB/archive/refs/tags/{{ vtag }}.tar.gz
-  sha256: e8e89c9c345415b17b30a2db3095ba9d47647611662073f7fbf54ad48b7f3c2a
+  sha256: 01598a46c1162c27253a0de0236f520fd8ee8166e9ebb84a4243574f88e6e50a
   patches:
-    - fix-build-gcc-13.3.patch
-    - 0001-Avoid-creating-python-eggs.patch
+    - 0001-fix-setup.patch
 
 build:
   number: 0
 
 requirements:
   build:
-    - python
+    - {{ stdlib('c') }}
     - {{ compiler('cxx') }}
-    # the ninja cmake generator only works on windows, for some reason
-    - ninja     # [win]
-    - m2-patch  # [win]
-    - patch     # [not win]
-    - make      # [not win]
     - cmake
+    - python
+    - ninja-base
     - pkg-config
 
 outputs:
-  - name: tbb
-    about:
-      summary: TBB Libraries
-      license_file:
-        - LICENSE.txt
-        - third-party-programs.txt
+  - name: {{ name }}
+    requirements:
+      build:
+        - {{ stdlib('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - python
+        - ninja-base
+        - pkg-config
```

```suggestion
        - pkg-config
      host:
        - libhwloc {{ libhwloc }}
```
From the logs, additional targets can be built when adding libhwloc.